releases/v1.6.0.toml

```diff
@@ -59,4 +59,15 @@ support to increase overall compatibility and stability.
 * **Allow WithServices to use custom implementations** ([#5709](https://github.com/containerd/containerd/pull/5709))
 * **Support custom compressor for walking differ** ([#5735](https://github.com/containerd/containerd/pull/5735))
 
+### Release Updates
+
+#### containerd/CNI/runc/critools tar bundle
+
+The tar bundles released as `cri-containerd-*.tar.gz` contain a build of runc
+linked with a newer version of libseccomp. This version of runc will not work on
+on some distributions, such as RHEL 7 and Debian 10. These tar bundles are now
+deprecated and will be removed or replaced in containerd 2.0. Projects relying
+on these tar bundles should use the `containerd-*.tar.gz` bundles or work with
+the containerd community on a suitable replacement in containerd 2.0.
```

Comment:
Could you update the deprecation list too
https://github.com/containerd/containerd/blob/v1.6.0-rc.4/RELEASES.md#deprecated-features